packages/phone-number-privacy/common/src/utils/authentication.ts

```diff
@@ -24,7 +24,7 @@ export async function authenticateUser(
   // https://tools.ietf.org/html/rfc7235#section-4.2
   const messageSignature = request.get('Authorization')
   const message = JSON.stringify(request.body)
-  const signer = request.body.account
+  const signer = JSON.stringify(request.body.account)
```

Comment:
Why was this necessary? Just wondering because in the case that this wasn't working before, we shouldn't take a dependency on this published version of pgpnp-common for this method.

Reply:
It probably wasn't necessary come to think of it. I was just trying to make sure that we didn't try and give bunyan something other than a string there but I don't think it matters